coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 0.0
+    "covered_percent": 29.17
```

Comment:
There are some weird files in there that we may or may not need, I'll look into it. Don't worry about the coverage right now